### PR TITLE
General

### PR DESCRIPTION
**Pull request by HadertS**
Navbar + Header + Footer setup

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		<meta http-equiv='x-ua-compatible' content='ie=edge'>
+		<meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>
+		<link rel='stylesheet' type='text/chref="index.html" ss' href='normalize.css'>
+		<link rel='stylesheet' type='text/css' href='styles.css'>
+		<title>Sammi Hadert</title>
+	</head>
+	<body>
+		<nav>
+			<h1>Sammi Hadert</h1>
+			<ul class="menu">
```
Comment (randallcoding) [inline index.html:14]: adjust text size and positioning

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
```
Comment (randallcoding) [inline styles.css:5]: Change font

**Review by randallcoding — APPROVED** [review]
generally font related comments, otherwise great work

File: index.html (hunk 1 above)
Comment (HadertS) [reply index.html:14]: will address in future version

**Pull request merged**